<launch>

    <rosparam param="stimulus/publish_voltages/channels_ai" >[0, 1, 2]</rosparam>
    


</launch>
